### ISSUE TITLE
Allow html in Product Description

### ISSUE THREAD

**Issue by rotexhawk** [reactioncommerce/reaction#2126]
It will be nice if we could add simple html tags like (`<br>`, `<li>`, `<p>` .etc) for product description, or maybe if the description field could just honor the spacing. 

Currently it removes all the spaces and converts the text into a simple string. Thank you.

**Comment by aaronjudd** [comment]
HTML in product descriptions would be easy enough to implement, but from a design standpoint I don't think it's an approach I would want to see in the core product, as I think it's better to enforce uniformity, and cleanliness in the data.  You (as an operator) may not know all the places where the description might be re-used.

**Comment by rotexhawk** [comment]
I agree, it is better to have clean data. My main goal for this ticket was to allow new lines. It is not unique to html and it is a part of natural language. Everything cramped up together makes the UI look bad.

**Comment by aaronjudd** [comment]
Agree, we should allow new lines.  We should not be striping spaces or new lines, etc. in saving the description.  (and I'm actually surprised if we are), so I'll re-open, rename to investigate this.

**Comment by mikemurray** [comment]
might just need to be put in a <pre> tag to preserve formatting.
